examples/llm/src/hf_causal_lm.py

```diff
@@ -1,30 +1,28 @@
 # Copyright 2022 MosaicML Examples authors
 # SPDX-License-Identifier: Apache-2.0
 
-from typing import Optional
-
 import torch
 import torch.nn.functional as F
 from composer.metrics.nlp import (HFCrossEntropy, LanguageCrossEntropy,
                                   Perplexity)
 from composer.models.huggingface import HuggingFaceModel
 from omegaconf import DictConfig
-from torch import Tensor
-from torchmetrics import Metric
 from transformers import AutoConfig, AutoModelForCausalLM, AutoTokenizer
+from transformers.modeling_outputs import CausalLMOutput
 
 from examples.common.hf_fsdp import prepare_hf_causal_lm_model_for_fsdp
 
 
 class ComposerHFCausalLM(HuggingFaceModel):
 
     def __init__(self, cfg: DictConfig):
-        config = AutoConfig.from_pretrained(cfg.hf_config_name_or_path)
-        tokenizer = AutoTokenizer.from_pretrained(cfg.hf_config_name_or_path)
+        config = AutoConfig.from_pretrained(cfg.pretrained_model_name_or_path)
```

Comment:
This class should really have a docstring that explains what the expected config fields are and what they mean.

Reply:
Punting to a future "Add Docstrings" PR